base_tier_validation/tests/test_tier_validation.py

```diff
@@ -9,7 +9,7 @@
 from odoo.tests.common import Form, tagged
 
 
-@tagged("post_install", "-at_install")
+@tagged("at_install")
 class TierTierValidation(common.SavepointCase):
```

Comment:
So, from this class, you will do 3:
CommonTestTierValidation - Will only contain the setUpClass and the Teardown
TestTierValidation that is post_install,  inherits CommonTestTierValidation and executes all tests except the view
TestTierValidationView that is at_install,  inherits CommonTestTierValidation and executes only the test view